### ISSUE TITLE
Can't join aliucord server

### ISSUE THREAD

**Issue by Lagiel1** [Aliucord/Aliucord#191]
### Tell us all about it.

This my new discord account and I tried to join the support server but could not as the invite button was not working. The invite dialogue box closes out.

### Request Agreement

- [X] I did indeed check to make sure the issue is original!

**Comment by Juby210** [comment]
What's your account, preferably id

**Comment by hairyfred** [comment]
> What's your account, preferably id

Including ANY accounts you have joined the server with, high chance you joined with a fresh account that got banned, causing discord to ip ban other accounts from joining

**Comment by Lagiel1** [comment]
Lagiel#4026

**Comment by Lagiel1** [comment]
I first tried to join the support server which I can't then tried other servers joined them without any problem

**Comment by Juby210** [comment]
Your account doesn't even have a day, we don't allow fresh accounts to join because they are most likely alts. Also you shouldn't know about discord mods if you have one day account.

**Comment by Lagiel1** [comment]
Ok so I can try again or is there no luck in this account

**Comment by Lagiel1** [comment]
Like tomorrow or a week

**Comment by hairyfred** [comment]
> Ok so I can try again or is there no luck in this account

Very unlikely this is your main account so the ban will stay
No one installs a discord mod after 1 day of using discord

**Comment by Lagiel1** [comment]
Ok

**Comment by hairyfred** [comment]
> Ok

So what happend to your old account?

**Comment by Lagiel1** [comment]
Server limited

**Comment by Lagiel1** [comment]
And the other reason being that i use alts on mod apps

**Comment by KiddieStranglersTM** [comment]
Commenting here to not clutter the issues panel.
I'm having essentially the same issue, tried to join with an alt so i wouldn't have to deal with bs on a public discord server because i despise public discord servers. After that didn't work i said fuck it and tried to join on my main account but I guess i got ip banned or something from that attempt and I'm also unable to join with my main account. 
Accounts:
Main - Trademarked®#2977 
Alt - Cringe#3053

**Comment by Juby210** [comment]
Unbanned.. Don't try to join with alt again

**Comment by den7007** [comment]
Hi @hairyfred @Juby210 , writing this here since it was [mentioned](https://github.com/Aliucord/Aliucord/issues/193#issuecomment-1013852970) to use an existing issue instead of creating a new one. I know this is closed but there was no existing open issue and I don't know how else to contact you guys. I can't join the support server even though this is my first/main Discord account.

It was mentioned earlier here "No one installs a discord mod after 1 day of using discord" and "Also you shouldn't know about discord mods if you have one day account" but I would like to kindly disagree. I value privacy greatly hence before signing up for and using all these popular social sites like Facebook, Instagram, Twitch, Discord etc. I do my research and ensure how to use them with the least amount of ads, trackers, analytics.. All of these services have their own modded/alternative apps which block or lessen the trackers/ads/bloat.. Of course it is better to not use them at all, but now I have decided after putting it off for so long to make a Discord account. So in my search before even making an account I already discovered some Discord desktop programs, browser add ons and mobile mods... I am sure you are aware of all these. As much as possible I try to use open source apps also hence I decided to go with Aliucord for my mobile phone.

TLDR: Me knowing about these apps/mods beforehand was because of privacy reasons (don't care much for the extra themes or emojis etc).

It's unfortunate the wanting for privacy is often interpreted as "sketchy" due to bad actors that ruin the experience for "innocent" folks like me but that is out of my control.. I don't know how else I can explain/prove that this is really my first discord account and not an "alt". 

Hoping you can consider unbanning my account so I can join the server. My account is: flaminghotcheeto # 0686  

Thank you!

**Comment by Vendicated** [comment]
why in the world did you type an essay lmao anyway unbanned

**Comment by den7007** [comment]
> why in the world did you type an essay lmao anyway unbanned

Hahaha, I just wanted to explain properly :-). Thanks. Are these bans instant or do I have to wait a while before trying again? I just tried joining using the app and it still can't seem to join. I'll try again later when I have access to my computer

**Comment by Vendicated** [comment]
The bot banned you again. you need to either wait 2 days or give me your id so I can whitelist you.

**Comment by den7007** [comment]
> The bot banned you again. you need to either wait 2 days or give me your id so I can whitelist you.

Got it, my ID 932133527879823380

**Comment by AnCopper** [comment]
Hey so, I got banned for posting a particular Kong copypasta, I'm sorry. Can I get unmanned? I won't do it again, my name and hash are F.F#3060

**Comment by DodoGames7** [comment]
this does look like a place to appeal?, no i don't think so

**Comment by hairyfred** [comment]
> 

This is not for ban appeals.
If you think you have been wrongly banned
Please fill in the form below
https://shorturl.at/rxE29

**Comment by ItzOnlyAnimal** [comment]
:trollface:

**Comment by sourcekilla** [comment]
![Screenshot_20220330-164212_Bromite](https://user-images.githubusercontent.com/94068062/160821815-fd9b1534-86b8-449b-b50b-635f9e538d81.jpg)
you want to?
